apps/docs/src/development/packages/changelogs/[name].md

```diff
@@ -0,0 +1,41 @@
+<script lang="ts" setup>
+import { useData } from "vitepress";
+import { ref } from "vue";
+
+const { params } = useData();
+</script>
+
+<div v-if="params.name === 'figma-utils'">
+
+<!--@include: @/../../../packages/figma-utils/CHANGELOG.md-->
+
+</div>
+
+<div v-else-if="params.name === 'headless'">
+
+<!--@include: @/../../../packages/headless/CHANGELOG.md-->
+
+</div>
+
+<div v-else-if="params.name === 'sit-onyx'">
+
+<!--@include: @/../../../packages/sit-onyx/CHANGELOG.md-->
+
+</div>
+
+<div v-else-if="params.name === 'storybook-utils'">
+
+<!--@include: @/../../../packages/storybook-utils/CHANGELOG.md-->
+
+</div>
+
+<div v-else-if="params.name === 'vitepress-theme'">
+
+<!--@include: @/../../../packages/vitepress-theme/CHANGELOG.md-->
+
+</div>
+
+<div v-else>
```

Comment:
Does VitePress have a default 404 page that we could route to?

Reply:
Yes there is one but I don't think we can automatically redirect to the `404.html` inside the `v-else` block.
I added a more detailed description to create a bug report instead :)